site/content/docs/0.0/utilities/api.md

```diff
@@ -487,12 +503,17 @@ This will now generate responsive variations of `.border` and `.border-0` for ea
 .border { ... }
 .border-0 { ... }
 
+@media (min-width: 390px) {
+  .border-xs { ... }
```

Comment:
With `$enable-bootstrap-compatibility`, the `.border-xs`, `.border-md`, etc. are not generated. Only the `*-0` versions. This is probably due to the modifications we've done for OUDS Web on the border side.
So either we fix this behavior for the borders, either we change here the sentence explaining what it's going to do + the examples of generated code.

Reply:
Give it a try with bourder-ouds